mathesar_ui/src/components/cell/data-types/email.ts

```diff
@@ -0,0 +1,12 @@
+import type { ComponentAndProps } from '@mathesar-component-library/types';
+import { TextInput } from '@mathesar-component-library';
+import type { CellComponentFactory } from './typeDefinitions';
+import TextBoxCell from './components/textbox/TextBoxCell.svelte';
+
+const emailType: CellComponentFactory = {
+  get: (): ComponentAndProps => ({ component: TextBoxCell }),
+
+  getInput: (): ComponentAndProps => ({ component: TextInput }),
+};
+
+export default emailType;
```

Comment:
This cell type isn't essential, since it's covered by a `string` cell.

Reply:
Thanks @pavish. This should be resolved by 43446c7768b33d027db1751a8dd49760020647dd.